css/properties/margin-left.json

```
@@ -4,6 +4,7 @@
       "margin-left": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-left",
+          "spec_url": "https://drafts.csswg.org/css-flexbox-1/#item-margins",
```

I think we should concentrate on getting the main specs for a feature.

```suggestion
          "spec_url": "https://drafts.csswg.org/css-box/#propdef-margin-left",
```

---

Should we link to sections or directly to the definitions? There are suggestions going on both directions now. https://drafts.csswg.org/css-box/#margin-physical would be the section here.

---

```suggestion
          "spec_url": "https://drafts.csswg.org/css-box/#margin-physical",
```